runtimes/js/encore.dev/storage/objects/bucket.ts

```diff
@@ -86,6 +86,19 @@ export class Bucket extends BucketPerms implements Uploader, Downloader, Attrser
     return unwrapErr(res);
   }
 
+  /**
+   * Generate an external URL to allow uploading an object to the bucket.
+   * 
+   * Anyone with possession of the URL can write to the given object name
+   * without any additional auth.
+   */
+  async getUploadUrl(name: string, options?: UploadUrlOptions): Promise<string> {
```

Comment:
Let's make this `signedUploadUrl`, and have it return a `SignedURL` object with just a single field `{url: string}`

```suggestion
  async signedUploadUrl(name: string, options?: SignedUploadUrlOptions): Promise<SignedURL> {
```

Reply:
Done